name,value,relevant log,explanation
io.bytes.per.checksum,0,"ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null
	",value-hit
io.bytes.per.checksum,0,"ERROR	Task: attempt_1701312551309_0082_r_000000_0 - exited : org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
io.bytes.per.checksum,0,"INFO	Diagnostics report from attempt_1701312551309_0082_r_000000_0: Error: org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
fs.s3a.fast.upload.active.blocks,4,INFO	1 failures on node ed761ad59d44,value-hit
mapreduce.map.maxattempts,4,INFO	1 failures on node ed761ad59d44,value-hit
